package.json

```diff
@@ -31,22 +31,22 @@
     "@capacitor/keyboard": "5.0.0",
     "@capacitor/status-bar": "5.0.0",
     "@capawesome/capacitor-file-picker": "5.0.0",
-    "@ionic/angular": "6.2.9",
-    "rxjs": "7.5.7",
+    "@ionic/angular": "7.1.2",
+    "rxjs": "7.8.1",
     "tslib": "2.4.0",
-    "zone.js": "0.11.8"
+    "zone.js": "0.13.1"
   },
   "devDependencies": {
-    "@angular-devkit/architect": "0.1402.4",
-    "@angular-devkit/build-angular": "14.2.4",
-    "@angular-eslint/builder": "14.1.2",
-    "@angular-eslint/eslint-plugin": "14.1.2",
-    "@angular-eslint/eslint-plugin-template": "14.1.2",
-    "@angular-eslint/template-parser": "14.1.2",
-    "@angular/cli": "14.2.4",
-    "@angular/compiler": "14.2.4",
-    "@angular/compiler-cli": "14.2.4",
-    "@angular/language-service": "14.2.4",
+    "@angular-devkit/architect": "0.1601.4",
+    "@angular-devkit/build-angular": "16.1.4",
+    "@angular-eslint/builder": "^16.0.0",
```

Comment:
Please keep the exact version numbers.

Reply:
I have done the update, please check again